src/components/navigation/problem-tab-content.tsx

```diff
@@ -44,9 +56,9 @@ export const ProblemTabContent: React.FC<IProps>
   return (
     <Tabs className={classNames("problem-tabs", context, chatBorder)} selectedTabClassName="selected"
           data-focus-document={problemPath}>
-      <div className="tab-header-row">
+      <div className={classNames("tab-header-row", { "no-sub-tabs": !hasSubTabs})}>
```

Comment:
```suggestion
      <div className={classNames("tab-header-row", {"no-sub-tabs": !hasSubTabs})}>
```
for style consistency with the surrounding code